chapter11/drjt/ex1.pm

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/perl
+use 5.010;
+use strict;
+use warnings;
+
+package ex1;
+
+use constant { DAY_NAMES => [qw( ark dip wap sen pop sep kir )],
+               MONTH_NAMES => [qw( diz pod bod rod sip wax lin sen kun fiz nap dep )],
+               };
+
+
+sub number_to_day_name {
+  my $num = shift;
+  return number_to_name( $num, "'$num' is not a valid day number!\n", DAY_NAMES );
+}
+
+sub number_to_month_name {
+  my $num = shift or die;
```

Comment:
Why the inconsistency between the months and days functions here? Do you want to `die` on parameters that resolve to false?